empire.go

```diff
@@ -79,6 +79,9 @@ type Empire struct {
 
 	// RunRecorder is used to record the logs from interactive runs.
 	RunRecorder RunRecorder
+
+	// MessagesRequired is a boolean used to determine if messages should be required for events.
+	MessagesRequired bool
```

Comment:
i would think requiring commit messages would be something that empire requires and the heroku client just knows how to handle. i'll take a look at this tomorrow.